jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -441,8 +441,9 @@
               export GINKGO_TEST_ARGS="--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]"
               export FAIL_ON_GCP_RESOURCE_LEAK="false"
               export PROJECT="k8s-jkns-pr-gce"
-              # NUM_NODES should match kubernetes-e2e-gce.
-              export NUM_NODES="3"
+              # NUM_NODES and GINKGO_PARALLEL_NODES should match kubernetes-e2e-gce.
```

Comment:
Yes, but I'd like to test it out and see what the performance improvements are first. If it only shaves off a couple minutes then it's probably not worth it.